priority/priority.erl

```diff
@@ -1,47 +1,45 @@
 -module(priority).
+
 -export([start/0,
          send_vip/2,
          send_normal/2]).
+
  -include_lib("eunit/include/eunit.hrl").
 
+start() ->
+    spawn(fun() -> priority_loop() end).
+
+send_vip(Pid, Msg) ->
+    Pid ! {vip, Msg}.
+
+send_normal(Pid, Msg) ->
+    Pid ! {normal, Msg}.
+
```

Comment:
consider adding a comment here signaling the start of private functions, like this https://github.com/lambdaclass/throttle/blob/master/src/throttle.erl#L51